index.bs

```diff
@@ -2096,103 +2132,72 @@ When this method is invoked, the user agent MUST execute the following algorithm
+                        interaction with a particular UI context
+                        (see [here](#GetAssn-ConditionalMediation-Interact-FormControl) for details).
 
-                        1. If |distinctTransports|
-                            <dl class="switch">
-                                :   [=list/is not empty=]
-                                ::  The client selects one |transport| value from |distinctTransports|, possibly incorporating local
-                                    configuration knowledge of the appropriate transport to use with |authenticator| in making its
-                                    selection.
-
-                                    Then, using |transport|, invoke the [=authenticatorGetAssertion=] operation on
-                                    |authenticator|, with |rpId|, |clientDataHash|, |allowCredentialDescriptorList|,
-                                    |userVerification|, and |authenticatorExtensions| as parameters.
-
-                                :   [=list/is empty=]
-                                ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|,
-                                    invoke the [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|,
-                                    |clientDataHash|, |allowCredentialDescriptorList|, |userVerification|, and
-                                    |authenticatorExtensions| as parameters.
-                            </dl>
+            1. Else:
 
-                    :   [=list/is empty=]
-                    ::  Using local configuration knowledge of the appropriate transport to use with |authenticator|, invoke the
-                        [=authenticatorGetAssertion=] operation on |authenticator| with |rpId|, |clientDataHash|,
-                        |userVerification| and |authenticatorExtensions| as parameters.
+                1. Execute the [=issuing a credential request to an authenticator=] algorithm with |authenticator|, |savedCredentialIds| and |options|.
 
-                        Note: In this case, the [=[RP]=] did not supply a list of acceptable credential descriptors. Thus, the
-                            authenticator is being asked to exercise any credential it may possess that is [=scoped=] to
-                            the [=[RP]=], as identified by |rpId|.
-                </dl>
+                    Note: This branch is taken if <code>|options|.{{CredentialRequestOptions/mediation}}</code> is {{CredentialMediationRequirement/conditional}}
+                    and the |authenticator| does not support the [=silentCredentialDiscovery=] operation to allow use of such authenticators during a
+                    {{CredentialMediationRequirement/conditional}} [=user mediation=] request.
 
             1. [=set/Append=] |authenticator| to |issuedRequests|.
```

Comment:
Looks like this should be indented into the preceding `Else` block? (Since the earlier "If options.mediation is conditional and the user interacts with an input form control with an autocomplete attribute whose value contains a "webauthn" autofill detail token" case appends to _issuedRequests_ at that time instead)

Reply:
By "this", if you are referring to L2202 "3. Append authenticator to issuedRequests.", then I am thinking you are correct.

Reply:
Good catch, you're right. Done.